lesson3/matrix_utils/sum_of_diagonals_or_diagonal_of_matrix.py

```diff
@@ -0,0 +1,40 @@
+from is_matrix_square import square_checker
+
+
+def sum_of_diagonals_of_matrix(matrix):
```

Comment:
should start from verb
```suggestion
def sum_diagonals_of_matrix(matrix):
```
or
```suggestion
def get_sum_of_diagonals_of_matrix(matrix):
```

Reply:
Done